include/podio/utilities/StaticConcatenate.h

```diff
@@ -0,0 +1,28 @@
+#ifndef PODIO_UTILITIES_STATICCONCATENATE_H
+#define PODIO_UTILITIES_STATICCONCATENATE_H
+
+#include <algorithm>
+#include <array>
+#include <string_view>
+namespace podio::utils {
+
+/// Helper struct to concatenate a set of string_views into a single string_view at compile time
+template <const std::string_view&... strs>
+struct static_concatenate {
+  static constexpr auto init_arr() {
+    constexpr auto total_size = (strs.size() + ... + 0);
+    auto arr = std::array<char, total_size>();
```

Comment:
This will result in a non null-terminated array of `char`, right?

Maybe 
```suggestion
    constexpr auto total_size = (strs.size() + ... + 1);
    auto arr = std::array<char, total_size>();
```
and then explicitly adding a `\0` as the last element of `arr` could be a good idea, just in case anything that needs to consume this (e.g. via `.data()`) expects a null-terminated string.

Reply:
Good idea